client/src/store/list-data/intl-jurisdictions.ts

```diff
@@ -1,1248 +1,1248 @@
 const jurisdictions = [
   {
     "value": "TM",
-    "SHORT_DESC": "TURKMENISTAN",
-    "text": "TURKMENISTAN"
+    "SHORT_DESC": "Turkmenistan",
+    "text": "Turkmenistan"
   },
   {
     "value": "TC",
-    "SHORT_DESC": "TURKS AND CAICO",
-    "text": "TURKS AND CAICOS ISLANDS"
+    "SHORT_DESC": "Turks And Caico",
+    "text": "Turks And Caicos Islands"
   },
   {
     "value": "TV",
-    "SHORT_DESC": "TUVALU",
-    "text": "TUVALU"
+    "SHORT_DESC": "Tuvalu",
+    "text": "Tuvalu"
   },
   {
     "value": "UG",
-    "SHORT_DESC": "UGANDA",
-    "text": "UGANDA"
+    "SHORT_DESC": "Uganda",
+    "text": "Uganda"
   },
   {
     "value": "UA",
-    "SHORT_DESC": "UKRAINE",
-    "text": "UKRAINE"
+    "SHORT_DESC": "Ukraine",
+    "text": "Ukraine"
   },
   {
     "value": "AE",
-    "SHORT_DESC": "UNITED ARAB EMI",
-    "text": "UNITED ARAB EMIRATES"
+    "SHORT_DESC": "United Arab Emi",
+    "text": "United Arab Emirates"
   },
   {
     "value": "GB",
-    "SHORT_DESC": "UNITED KINGDOM",
-    "text": "UNITED KINGDOM"
+    "SHORT_DESC": "United Kingdom",
+    "text": "United Kingdom"
   },
   {
     "value": "UM",
-    "SHORT_DESC": "UNITED STATES M",
-    "text": "UNITED STATES MINOR OUTLYING ISLANDS"
+    "SHORT_DESC": "United States M",
+    "text": "United States Minor Outlying Islands"
   },
   {
     "value": "UY",
-    "SHORT_DESC": "URUGUAY",
-    "text": "URUGUAY"
+    "SHORT_DESC": "Uruguay",
+    "text": "Uruguay"
   },
   {
     "value": "ME",
-    "SHORT_DESC": "MONTENEGRO",
-    "text": "MONTENEGRO"
+    "SHORT_DESC": "Montenegro",
+    "text": "Montenegro"
   },
   {
     "value": "RS",
-    "SHORT_DESC": "SERBIA",
-    "text": "SERBIA"
+    "SHORT_DESC": "Serbia",
+    "text": "Serbia"
   },
   {
     "value": "IM",
-    "SHORT_DESC": "ISLE OF MAN",
-    "text": "ISLE OF MAN"
+    "SHORT_DESC": "Isle Of Man",
+    "text": "Isle Of Man"
   },
   {
     "value": "GG",
-    "SHORT_DESC": "GUERNSEY",
-    "text": "GUERNSEY"
+    "SHORT_DESC": "Guernsey",
+    "text": "Guernsey"
   },
   {
     "value": "JE",
-    "SHORT_DESC": "JERSEY",
-    "text": "JERSEY"
+    "SHORT_DESC": "Jersey",
+    "text": "Jersey"
   },
   {
     "value": "BQ",
-    "SHORT_DESC": "BONAIRE",
-    "text": "BONAIRE"
+    "SHORT_DESC": "Bonaire",
+    "text": "Bonaire"
   },
   {
     "value": "CW",
-    "SHORT_DESC": "CURACAO",
-    "text": "CURACAO"
+    "SHORT_DESC": "Curacao",
+    "text": "Curacao"
   },
   {
     "value": "SX",
-    "SHORT_DESC": "ST MAAR DP",
-    "text": "SINT MAARTEN(DUTCH PART)"
+    "SHORT_DESC": "St Maar Dp",
+    "text": "Sint Maarten(dutch Part)"
```

Comment:
Capital D utch.

Reply:
And on line 84, Dp should be DP (since it is initials for Dutch Part).

Reply:
oh i missed those. let me go through that